router/core/graphql_prehandler.go

```diff
@@ -609,6 +610,15 @@ func (h *PreHandler) handleOperation(req *http.Request, variablesParser *astjson
 	}
 	requestContext.telemetry.addCommonAttribute(attributesAfterParse...)
 
+	if isBatchedRequest, _ := req.Context().Value(batch.IsBatchedRequestKey{}).(bool); isBatchedRequest {
+		if requestContext.operation.opType != "query" {
+			return &httpGraphqlError{
+				message:    "Batched requests can only contain queries",
```

Comment:
Is this the same response as apollo? Additionally, will this error be tracked in metrics / tracing?

Reply:
Apollo allows mutations, we don't; so this will be a new error. I would suggest we just allow queries for now and allow mutations if customers request it (as its harder to go from permissive -> restrictive after its out in the wild).
![image](https://github.com/user-attachments/assets/3c61d749-a2e1-48f3-9754-fa9b6ae401dc)

Reply:
It allows mutations, we don't; so this will be a new error. I would suggest we just allow queries for now and allow mutations if customers request it (as its harder to go from permissive -> restrictive after its out in the wild).

Reply:
If it supports mutation by spec, we need to support it because we're building as a migration feature.